tests/about-us.spec.ts

```diff
@@ -1,14 +1,30 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, type Page } from '@playwright/test';
 
 test.beforeEach(async ({ page }) => {
 	await page.goto('/about-us');
 });
 
+async function forceLoadImages(page: Page) {
```

Comment:
This function is pretty cool. Probably useful for the other tests, too.

Reply:
@Han2-Ro I think so too. Maybe create a follow-up issue to clean up the tests and make them use this function?